CHANGELOG.md

```diff
@@ -12,5 +12,6 @@ All notable, unreleased changes to this project will be documented in this file.
 
 ### Other changes
 - Improved page search with search vectors. Pages can now be searched by slug, title, content, attribute values, and page type information.
+- Added new mount point for App Extensions: `TRANSLATION_DETAILS`
```

Comment:
If you want to port i to 3.22, then the changelog is not needed as it is for 3.23